drv/lpc55-swd/src/main.rs

```diff
@@ -512,28 +613,174 @@ impl idl::InOrderSpCtrlImpl for ServerImpl {
         _msg: &userlib::RecvMessage,
     ) -> Result<(), RequestError<core::convert::Infallible>> {
         ringbuf_entry!(Trace::DisabledWatchdog);
+        self.watchdog_ms = None;
```

Comment:
I don't understand the point of `watchdog_ms`; as far as I can tell, it is written but never used?

Reply:
You are correct. It isn't needed. It is left over from me tracking the interaction of dump, watchdog, and measurement activities.